### ISSUE TITLE
Swipe and AppBar Actions doesnt work well. 

### ISSUE THREAD

**Issue by oswaldocueto01** [Dn-a/flutter_inner_drawer#54]
When i use Swipe property i can tap all my icon button widget. exists some space or with near to the device side that cause the problem.

![image](https://user-images.githubusercontent.com/64932566/98424472-cbe9a100-205f-11eb-9bf4-ce8d847728c1.png)

![image](https://user-images.githubusercontent.com/64932566/98424522-ede32380-205f-11eb-856d-a622c1323327.png)

I cant touch all this area...

**Comment by oswaldocueto01** [comment]
Update... i fixed it changing the 
`const double _kEdgeDragWidth = 20.0;`
to
`const double _kEdgeDragWidth = 0.0;`

thats all, Thanks you

**Comment by oswaldocueto01** [comment]
Consider to change that constant to a public for change it before init state or implement it as a parameter.

**Comment by noobloser** [comment]
tl;dr: definition of a swipe area

If possible the definition of a designated swipe area might be beneficial.
Having a Listview with an overlay button to open the sidebar, the user might also like to swipe.
If the items also have a swipe action attached to them it might get complicated, so limiting the flutter_inner_drawer swipe area to the button area might help here.
![Screenshot from 2020-11-30 03-43-55](https://user-images.githubusercontent.com/25504634/100563296-5d92aa00-32be-11eb-9596-ce6982068de6.png)
